CONTRIBUTING.md

```diff
@@ -58,32 +58,22 @@ by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
     - Please note that you PR should be directed from your branch (for example `myfeature`) towards the branch `dev`
+    - To make reviewing easier, briefly describe the changes you have made in the pull request and summarise the discussion and conclusions in the associated issue. 
 7. Describe briefly (i.e. in one or two lines) what you changed in the `CHANGELOG.md` file. End the description by the number in parenthesis `(#<your PR number>)`
 8. Commit the changes to the `CHANGELOG.md` file
-9. Write the PR number in the corresponding issue so that they are linked. Write it with one of the [special keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) so that the issue will be automatically closed when the PR is merged (example: `Closes #<your issue number>`)
-10. [Ask](https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users) for review of your PR 
- 
-    A) If including new terms to the ontology, ask at least two persons to review
-    
-    B) If restructuring the ontology, ask at least two persons to review
-    
-    C) If updating a definition in the ontology, ask at least two
-    persons to review
-    
-    D) If your PR does not modify the ontology, one reviewer is
-    enough
-
-    When more than one reviewer is requested, please try to combine modelling and ontology expertise in your choice of reviewers. (see the section "Teams tag" of the [README](https://github.com/OpenEnergyPlatform/ontology/blob/dev/README.md) for more information about who is expert of what)
+9. Write the corresponding issue number in the pull request so that they are linked. Write it with one of the [special keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) so that the issue will be automatically closed when the PR is merged (example: `Closes #<your issue number>`)
+10. [Ask](https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users) for review of your PR.  
+    As the issue will have been discussed and agreed on prior to implementation, the purpose of the review step post-implementation is to check that the implementation has been faithful to what was agreed. One or two reviewers may be needed depending on the nature of the change that has been made. If the change involves adding content (A), a domain expert should review the issue. If the change involves restructuring the ontology (B), an ontology expert should review. If the change involves both changes to content and restructuring (B and C), it is best to ask both an ontology expert and a domain expert to review. See the section "Teams tag" of the [README](https://github.com/OpenEnergyPlatform/ontology/blob/dev/README.md) for more information about the expertise of the different team members.
```

Comment:
We are talking about 'ontology experts' and 'domain experts' here, which makes sense to me. What about integrating the existing teams 'modelling experts' and 'economic experts' into just one team 'domain experts'?

Reply:
I had the impression, that along with restructuring, there is also always a content change. Mostly because a change in structure has impact on the definitions. That is why I proposed two reviewers in the first place.